integration-libs/opf/b2b-checkout/components/opf-b2b-checkout-cost-center/opf-b2b-checkout-cost-center.component.html

```diff
@@ -0,0 +1,30 @@
+<ng-container *ngIf="isAccountPayment">
+  <div class="row">
+    <div class="col-md-12 col-xl-10">
+      <ng-container *ngIf="costCenters$ | async as costCenters">
+        <div *ngIf="costCenters.length !== 0">
+          <label>
+            <span class="label-content required">{{
+              'checkoutB2B.costCenter' | cxTranslate
+            }}</span>
+            <select
+              #costCenterSelect
+              (change)="setCostCenter(costCenterSelect.value)"
```

Comment:
Why use a component reference `costCenterSelect` instead of `$event.target.value`?

Reply:
In a previous approach I'm getting a linter error: 'Property 'value' does not exist on type 'EventTarget'.
Using a template reference Angular can properly infer the type of the element.